metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -211,6 +219,12 @@ object ServerCommands {
     CancelCompile,
     BspSwitch,
     StartDebugAdapter,
-    NewScalaFile
+    NewScalaFile,
+    ResolveDebugClass
   )
 }
+
+case class DebugClassParams(
+    mainClass: String,
+    @Nullable project: String = null
```

Comment:
For better or worse we should probably use “build target” instead of “project” here

Reply:
but will that be clear for users, what it is?